dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -66,7 +66,7 @@ private DynatraceMetricExporter(
       URL url,
       String apiToken,
       String prefix,
-      Labels defaultDimensions,
+      Attributes defaultDimensions,
       Boolean enrichWithOneAgentMetaData) {
```

Comment:
Not part of this PR, but is there a reason this is a boxed Boolean? Could throw NullPointerException when used in expressions when it is `null`.

Reply:
No idea tbh. Maybe @pirgeo knows?

Reply:
TBH, I am not sure. Probably just an oversight on my part, you can try and replace it and see if something breaks 😇

Reply:
I changed to use the value type one. Everything still works.